WooCommerce/Classes/ViewRelated/Top Banner/TopBannerPresenter.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit
+
+/// Ability to show and hide a banner
+///
+protocol TopBannerPresenter {
```

Comment:
Sorry if this is a stupid question, but where is this protocol consumed?

Reply:
ah my bad, it's consumed in https://github.com/woocommerce/woocommerce-ios/pull/1247